src/editor.js

```diff
@@ -1500,16 +1657,20 @@ export class OdooEditor extends EventTarget {
     }
 
     /**
-     * Prevent the pasting of HTML and paste text only instead.
+     * Handle safe pasting of html or plain text into the editor.
      */
     _onPaste(ev) {
         ev.preventDefault();
-        const pastedText = (ev.originalEvent || ev).clipboardData.getData('text/plain');
-        this.execCommand('insertText', pastedText);
+        const clipboardData = ev.clipboardData.getData('text/html');
+        if (clipboardData) {
+            this.execCommand('insertHTML', this._prepareClipboardData(clipboardData));
+        } else {
+            const text = ev.clipboardData.getData('text/plain');
+            this.execCommand('insertHTML', this._prepareTextClipboardData(text));
+        }
     }
-
     /**
-     * Prevent the dropping of HTML and paste text only instead.
+     * Handle safe dropping of html into the editor.
```

Comment:
I noticed that you can't seem to actually be able to drop plain text. Whatever I dropped was always html. Text just won't drop.